src/Adapter/PlatformServices.Shared/netstandard1.0/Services/ns10TestContextPropertyStrings.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices
+{
+    #pragma warning disable SA1649 // File name must match first type name
+    
+    internal static class TestContextPropertyStrings
```

Comment:
Consider adding this to Constants.cs file itself and not create a seperate file.

Reply:
In Full framework also we have followed this pattern hence kept it consistent here.